editors/code/src/util.ts

```diff
@@ -1,22 +1,41 @@
 import * as lc from "vscode-languageclient";
 import * as vscode from "vscode";
+import { strict as nativeAssert } from "assert";
+
+export function assert(condition: unknown, explanation: string): asserts condition {
```

Comment:
I think a single `assert(condition: boolean, explanation: string)` should be enough.

Reply:
Sure!

Reply:
Sorry, do you mean `boolean`? If yes, we would need to explicitly use null comparisongs `assert(obj == null)` vs `assert(obj)`, or is this desired?

Reply:
Yeah, I think explicit boolean is good

Reply:
Done!